### DISCUSSION TITLE
Error in repository setup: `cli-table` not found in the registry

### DISCUSSION THREAD

**Discussion by maksos070220** [NativeScript/NativeScript#10064] [General]
### Issue Description

![image](https://user-images.githubusercontent.com/64012463/197417488-f09a6e97-0644-45f1-970b-038816c62c31.png)

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by rigor789** [comment]
The url seems fine: https://github.com/telerik/cli-table/tarball/v0.3.1.2
But your logs indicate npm is somehow trying to load: https://dns.google/telerik/cli-table/tarball/v0.3.1.2

```diff
- https://github.com/telerik/cli-table/tarball/v0.3.1.2
+ https://dns.google/telerik/cli-table/tarball/v0.3.1.2
```

I believe this is an issue with your local DNS setup, however this is not related to NativeScript/this repo specifically...

Try resetting your npm registry:
```
npm config set registry http://registry.npmjs.org
```
